src/core/utils/request/getRequestCookies.ts

```diff
@@ -1,8 +1,21 @@
-import cookieUtils from '@bundled-es-modules/cookie'
+import * as cookieUtils from 'cookie'
 import { cookieStore } from '../cookieStore'
 
+function parseCookies(input: string): Record<string, string> {
+  const parsedCookies = cookieUtils.parse(input)
+  const cookies: Record<string, string> = {}
+
+  for (const cookieName in parsedCookies) {
+    if (typeof parsedCookies[cookieName] !== 'undefined') {
```

Comment:
Account for `cookie@1.0` annotating the parsing result as `Record<string, string | undefined>`. Skip the cookies with undefined values.